pkgs/top-level/all-packages.nix

```diff
@@ -34884,15 +34884,16 @@ with pkgs;
 
   transcribe = callPackage ../applications/audio/transcribe { };
 
-  transmission = callPackage ../applications/networking/p2p/transmission { };
+  transmission_4 = callPackage ../applications/networking/p2p/transmission { };
   transmission_3 = callPackage ../applications/networking/p2p/transmission/3.x.nix { };
-  libtransmission = transmission.override {
+  libtransmission_4 = transmission.override {
     installLib = true;
     enableDaemon = false;
     enableCli = false;
   };
   transmission-gtk = transmission.override { enableGTK3 = true; };
   transmission-qt = transmission.override { enableQt = true; };
+  libtransmission = transmission_3;
```

Comment:
This should probably go to `aliases.nix`. The only consumer, torrential already uses `transmission_3` directly.